shared/utils/routeHelpers.js

```diff
@@ -72,3 +72,7 @@ export function signin(service: string = 'slack'): string {
 export function settings(): string {
   return `/settings`;
 }
+
+export function groupSettings(): string {
```

Comment:
How about making the subpage an optional prop for `settings` ?

Reply:
tried that first, but seemed too complex, from a DSL standpoint, this seemed better until we have a couple of more instances